express_relay/sdk/js/src/examples/SimpleSearcher.ts

```diff
@@ -1,9 +1,67 @@
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
-import { checkHex, Client } from "../index";
+import { BidStatusUpdate, checkHex, Client, Opportunity } from "../index";
 import { privateKeyToAccount } from "viem/accounts";
 import { isHex } from "viem";
 
+const DAY_IN_SECONDS = 60 * 60 * 24;
+
+class SimpleSearcher {
+  constructor(
+    public client: Client,
+    public chainId: string,
+    public privateKey: string
+  ) {
+    this.client.setBidStatusHandler(this.bidStatusHandler.bind(this));
+    this.client.setOpportunityHandler(this.opportunityHandler.bind(this));
```

Comment:
I still think that this is not a non-intutive API and recommend moving it to the constructor of Client.